docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -131,6 +131,29 @@ each vulnerability. The rating indicates how important an update is:
 | **Moderate**  | Releases of moderate severity should be applied as soon as possible. They allow the unauthorized editing or creation of content. *Examples: Cross Site Scripting (XSS) in template helpers.* |
 | **Low**       | Low risk releases fix information disclosure and read-only privilege escalation vulnerabilities. These updates should also be applied as soon as possible, but with an impact-dependent priority. *Example: Exposure of the core version number, Cross Site Scripting (XSS) limited to the admin interface.* |
 
+### Internal Security Process
+
+Follow these instructions in sequence as much as possible:
+
+ * When receiving a report:
+   * Perform initial criticality assessment
```

Comment:
`, and ensure that the reporter is given a justification for all issues we classify or demote as non-security vulnerabilities`.